incident/utils/charts.py

```diff
@@ -71,7 +71,7 @@ def branches(self):
         return json.dumps(branches_value)
 
 
-class TreeMapBranches(TextChoices):
+class IncidentBranches(TextChoices):
```

Comment:
It does occur to me here that I chose the name "branches" for this concept because of how it related to the "tree" map chart, so I hope this terminology is still somewhat meaningful when applied to other visualizations. :sweat_smile: